lib/src/Uncertainty/Algorithm/Simulation/openturns/PhysicalSpaceCrossEntropyImportanceSampling.hxx

```diff
@@ -0,0 +1,93 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief PhysicalSpaceCrossEntropyImportanceSampling implements Importance Sampling by Cross Entropy algorithm in Physical Space
+ *
+ *  Copyright 2005-2023 Airbus-EDF-IMACS-ONERA-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#ifndef OPENTURNS_PHYSICALSPACECROSSENTROPYIMPORTANCESAMPLING_HXX
+#define OPENTURNS_PHYSICALSPACECROSSENTROPYIMPORTANCESAMPLING_HXX
+
+
+#include "openturns/CrossEntropyImportanceSampling.hxx"
+#include "openturns/OptimizationAlgorithm.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+/**
+ * @class PhysicalSpaceCrossEntropyImportanceSampling
+ *
+ * PhysicalSpaceCrossEntropyImportanceSampling is some PhysicalSpaceCrossEntropyImportanceSampling type to illustrate how to add some classes in OpenTURNS
+ */
+
+
+class OT_API PhysicalSpaceCrossEntropyImportanceSampling
+  : public CrossEntropyImportanceSampling
+{
+CLASSNAME
+public:
+/** Default constructor */
+  PhysicalSpaceCrossEntropyImportanceSampling();
+
+  /** Default constructor */
+  PhysicalSpaceCrossEntropyImportanceSampling(const RandomVector & event,
+                                                         const Point &activeParameters,
+                                                         const Point &initialAuxiliaryDistributionParameters,
+                                                         const Interval &bounds,
+                                                         const Distribution &auxiliaryDistribution,
+                                                         const Scalar rhoQuantile = ResourceMap::GetAsScalar("PhysicalSpaceCrossEntropyImportanceSampling-DefaultRhoQuantile"));
```

Comment:
Actually, in practice, the case where all the active parameters are optimized is not recommended. Often, we prefer to optimize only the means of the auxiliary distribution.

Reply:
ok
just maybe the auxiliaryDistribution argument should be moved before the activeParameters ?

Reply:
You're right. This has been fixed

Reply:
@m-balesdent If a discussion is over (because the problem has been fixed), could you please mark is as "solved"? This makes it easier to navigate the GitHub thread.